Frameworks/CoreGraphics/CGContextCairo.mm

```diff
@@ -2002,31 +1997,17 @@
     CGContextStrokePath();
 
     // Lock the data that is backing the layer.
-    BYTE* buffer = static_cast<BYTE*>(_imgDest->Backing()->LockImageData());
-    THROW_NS_IF_NULL(E_UNEXPECTED, buffer);
-    int stride = _imgDest->Backing()->BytesPerRow();
-    float width = _imgDest->Backing()->Width();
+    _imgDest->Backing()->LockImageData();
```

Comment:
I don't think that is correct.  The render target is created once, right? (It should be created only once).  The idea here is that when the d2d render target wants to draw, it will call wicbitmap lock - which should be used to lock the data buffer.  

---
In reply to: [80797038](https://github.com/Microsoft/WinObjC/pull/1040#discussion_r80797038) [](ancestors = 80797038,80771507)